webapi/src/main/scala/org/knora/webapi/responders/admin/UsersResponderADM.scala

```diff
@@ -260,30 +260,15 @@ class UsersResponderADM(responderData: ResponderData) extends Responder(responde
             )
 
             // check if we want to change the email
-            _ = if (changeUserRequest.email.isDefined) {
```

Comment:
This line throws away the `Future` returned by `userByEmailExists`. The failure containing the exception is inside that `Future`, so if you throw away the `Future`, the exception can't be propagated. You have to "unpack" the `Future`, using `<-` as you do in this PR, to propagate the exception.

Reply:
ok thanks. I still have no idea how you would do that, so I found another solution.

Reply:
The way you did it looks fine. Once this PR is done I'll be glad to review it.

Reply:
great. thanks for your help. It should be good now.
Is the failing ResourcesRouteV2E2E a common issue in the CI?